plugin/pkg/scheduler/factory/plugins.go

```diff
@@ -73,6 +101,32 @@ func RegisterPriorityFunction(name string, function algorithm.PriorityFunction,
 	return name
 }
 
+// RegisterCustomPriority registers a custom priority function with the algorithm registry.
```

Comment:
This should say RegisterCustomPriorityFunction, but I'd suggest the pattern I described earlier of not redundantly mentioning the name in the comment.